zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -2011,7 +2011,33 @@ paths:
                 $ref: "#/components/schemas/ProblemDetail"
         "500":
           $ref: "#/components/responses/InternalServerError"
-
+    delete:
+      tags:
+        - Role
+      operationId: deleteRole
+      summary: Delete role
+      description: Deletes the role with the given key.
+      parameters:
+        - name: roleKey
+          in: path
+          required: true
+          description: The key of the role to delete.
+          schema:
+            type: integer
+            format: int64
+      responses:
+        "201":
```

Comment:
I'll change it to 204 because that's what we actually return right now, but I believe we should change the API to never return a 204 as we want to include a response body and https://www.rfc-editor.org/rfc/rfc9110#status.204 says:
> A 204 response is terminated by the end of the header section; it cannot contain content or trailers.

Reply:
I agree :) - we can follow up with that as the pattern is used in a couple of places AFAIK